operator/config/crd/bases/topology.node.k8s.io_noderesourcetopologies.yaml

```diff
@@ -0,0 +1,270 @@
+apiVersion: apiextensions.k8s.io/v1
+kind: CustomResourceDefinition
+metadata:
+  annotations:
+    api-approved.kubernetes.io: https://github.com/kubernetes/enhancements/pull/1870
+    controller-gen.kubebuilder.io/version: v0.11.2
+  creationTimestamp: null
+  name: noderesourcetopologies.topology.node.k8s.io
+spec:
```

Comment:
Don't we have a copy of this already in the repo ? Then we probably shouldn't add this again, just use a single copy.